config/xml_schemas/config_batch.xsd

```diff
@@ -57,9 +58,12 @@
         <xs:element minOccurs="0" ref="jobid_pattern"/>
 
   <!-- depend_string: How to indicate to batch system that this job should not run until
-       a previous job has completed successfully -->
+       a previous job has completed (possibly unsuccessfully) -->
```

Comment:
No; I've been asking; but didn't get a response. Is this an okay thing to do? I can make a more complicated change and add a depend_any string.